drivers/wifi/winc1500/Kconfig.winc1500

```diff
@@ -15,6 +15,12 @@ config WIFI_WINC1500_NAME
 	string "Driver name"
```

Comment:
Please change the commit subject to `drivers: wifi: winc1500: Add new config flag`
Also no need to put so many empty lines to the commit message.
So the commit message should look something like this:
```
drivers: wifi: winc1500: Add new config flag

There is a hardcoded macro in Winc1500 HAL SPI driver
that would prevent the driver from working.
This macro is now defined only based on KConfig entry.
This KConfig entry is by default not set.
To enable, set "CONFIG_WINC1500_DRV_USE_OLD_SW=y"
in proj.conf or board.conf

Signed-off-by: Raja D.Singh <rdsingh@iotwizards.com>
```

Reply:
> Please change the commit subject to `drivers: wifi: winc1500: Add new config flag`
> Also no need to put so many empty lines to the commit message.
> So the commit message should look something like this:
> 
> ```
> drivers: wifi: winc1500: Add new config flag
> 
> There is a hardcoded macro in Winc1500 HAL SPI driver
> that would prevent the driver from working.
> This macro is now defined only based on KConfig entry.
> This KConfig entry is by default not set.
> To enable, set "CONFIG_WINC1500_DRV_USE_OLD_SW=y"
> in proj.conf or board.conf
> 
> Signed-off-by: Raja D.Singh <rdsingh@iotwizards.com>
> ```

OK.

Reply:
Done.